cmake/options.cmake

```diff
@@ -34,10 +34,12 @@ endif()
 
 option(BUILD_TESTS          "Build tests"                                                   OFF)
 option(ENABLE_CHECK_INTERNAL "Enable internal checks"                                       OFF)
+option(ENABLE_OSS_FUZZ      "Enable the OSS-Fuzz related targets"                           ON)
 option(BUILD_GUI            "Build the qt application"                                      OFF)
 option(WITH_QCHART          "When building GUI(need BUILD_GUI=ON), use Qt5 Charts"          OFF)
 
 option(HAVE_RULES           "Usage of rules (needs PCRE library and headers)"               OFF)
+option(USE_Z3               "Usage of z3 library"                                           OFF)
```

Comment:
I will start recommending that Cppcheck is built with Z3. If "OFF" here means the default value I'd suggest that it's changed to "ON".

Reply:
Alright.

Reply:
Since this would also be used on Windows where the external dependencies are not available by default I would leave it OFF like default.

Reply:
ok